### ISSUE TITLE
Change the `renovate/stability-days` status check to `renovate/minimum-release-age`

### ISSUE THREAD

**Issue by HonkingGoose** [renovatebot/renovate#21442]
### Describe the proposed change(s).

Creating an issue to track this idea.

### Original comments leading to this issue

> > Document that the minimumReleaseAge config option causes a renovate/stabilityDays option to be added. This would be very unintuitive to new users of the feature otherwise.
> 
> Is changing the `renovate/stability-days` status check to `renovate/minimum-release-age` also planned? I know that it would have to be done separately as it would be a breaking change

Planned but I don't think we have an issue yet. I would like to think if it's the right time to let users customize Renovate's status check strings too

_Originally posted by @rarkins in https://github.com/renovatebot/renovate/issues/21376#issuecomment-1504727770_

### Todos:

- [ ] #21376
- [ ] Change the `renovate/stability-days` status check to `renovate/minimum-release-age`
- [ ] _Optional:_ let users customize Renovate's status check strings

**Comment by HonkingGoose** [comment]
@rarkins want me to split the "customize Renovate's status check strings" thing into it's own issue? That way we can refactor the name in one issue, and improve the feature in another issue. 😄

**Comment by rarkins** [comment]
Yes, good idea

**Comment by HonkingGoose** [comment]
@rarkins can you explain _why_ you labeled this issue `status:blocked`? I read #22704, but that does not explain things for me.

**Comment by rarkins** [comment]
Blocked by #19980
